src/common/Cryptography/HMACSHA1.h

```diff
@@ -19,27 +19,76 @@
 #define _AUTH_HMAC_H
 
 #include "Define.h"
+#include <array>
 #include <string>
 #include <openssl/hmac.h>
 #include <openssl/sha.h>
 
 class BigNumber;
 
-#define SEED_KEY_SIZE 16
-
-class TC_COMMON_API HmacHash
+namespace Trinity
+{
+namespace Crypto
+{
+class TC_COMMON_API HMAC_SHA1
```

Comment:
I don't like how this gets a refactor going in a different direction than in master branch

https://github.com/TrinityCore/TrinityCore/blob/b3ba029791a9003880d77a048b16e9e593485fe9/src/common/Cryptography/HmacHash.h#L49-L50